pages/index.tsx

```diff
@@ -62,10 +66,40 @@ export default function Home() {
             ))}
           </ButtonGroup>
         </SectionContainer>
+        <SectionContainer sx={{ margin: 'auto', maxWidth: '65rem' }} maxWidth={false}>
+          <Box>
+            <Grid container>
+              <Grid item xs={12}>
+                <Typography variant="h2" sx={{ textAlign: 'center' }}>
+                  {content.ourMission.header}
+                </Typography>
+              </Grid>
+              <Grid item sm={12} md={6}>
+                <ImageContainer
+                  src={content.ourMission.imgsrc as string}
+                  alt=""
+                  width={406}
+                  height={306}
+                  style={{ width: '100%' }}
+                  useImageDimensions
+                />
+              </Grid>
+              <Grid item sm={12} md={6}>
+                <Typography variant="subtitle1" sx={{ mt: '4.375rem', textAlign: 'center' }}>
```

Comment:
Text align here should only be center for xs and sm, on desktops it should be left aligned. See `HeroBanner.tsx` line 99 for an example of how to use different sizes within `sx`.

Reply:
Updated